WTWSMS/common/religions/00_religions.txt

```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```

Comment:
Also, I find it odd they'd ever use a bible as justification when for several centuries the Irish took pride in being insulars. The "Romanii", a faction of Irish monks in the 800's who advocated following the Roman Catholics a lot more, wanted to put an end to incestuous marriage between cousins, and the Irish told them to, well, fuck off.